sub-packages/bionemo-geneformer/src/bionemo/geneformer/run/config_models.py

```diff
@@ -116,6 +116,7 @@ def construct_data_module(self, global_batch_size: int) -> SingleCellDataModule:
             persistent_workers=self.num_dataset_workers > 0,
             pin_memory=False,
             num_workers=self.num_dataset_workers,
+            bypass_tokenizer_vocab=True,
```

Comment:
do you think it makes sense to expose this in the config class? or would that add too much noise?

Reply:
Docstring of this class needs to be updated, it references `sc_memmap`, might make sense to do a search over the whole repo for references to `sc_memmap`

Reply:
@skothenhill-nv @jstjohn  I assumed that since the data will probably have tokens not in the HuggingFaceTokenizer Vocab, it would be fine to pass it in as True, but can add it in if you think it would be helpful. Also, in general, do we want our default behavior to be bypass_tokenizer_vocab=True (instead of bypass_tokenizer_vocab=False), even when we do expose the API to make it user configurable?  @polinabinder1  was in favor of using ypass_tokenizer_vocab=True.